src/manage-apps/manage-repos-manually.md

```diff
@@ -0,0 +1,167 @@
+# Directly Editing Repository Configuration Files
+
+Add and remove repositories by directly editing text files located in `/etc/apt/sources.list.d/` (Debian repositories) or `~/.local/share/flatpak/repo/` (Flatpak repositories).
+
+## Manually Adding Debian Debian Repositories
+
+In order to manually create a text file for a repository, it's important to understand the deb822 format. This format organizes a repository's attributes into human-readable fields.
+
+| Field | Function |
+|-------|----------|
+| X-Repolib-Name | Define the name of the source. This is used by Repoman in the `Source` column of the `Extra Sources` tab.|
+| Enabled | Enable or disables the repository.|
+| Types | Define the source file or code used to install packages.|
+| URIs | Provide the repository's address.|
+| Suites | Add the sources used to update packages. This field corresponds with the toggle switches in the `Updates` tab in Repoman.|
+| Components | Specify if the source should include software that is open-source, closed-source, officially supported by canonical, or community-maintained. This option corresponds with the toggle switches in the `Settings` tab in Repoman.|
```

Comment:
Canonical is a proper noun (company name), and we want to capitalize that out of respect for them and grammatical correctness.